### DISCUSSION TITLE
Issue in Sort Order in Popup List

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#14284] [How to] [answered]
Hello,

I want to sort list shown in Popup while selecting project, I tried setting Sort Order in Filters in Settings, but no success. I checked on Gitdeveloper and found same issue. Please assist how to default sort the list shown in Popup

![image](https://user-images.githubusercontent.com/12723595/108964832-007e7880-76a2-11eb-86bb-54c96f1f115b.png)

This list does not sort
![image](https://user-images.githubusercontent.com/12723595/108965025-49363180-76a2-11eb-93c2-0e1ffceeeced.png)

Thanks in advance
Regards
Jeevan Pendhari

**Comment by vovpff** [comment, marked as answer]
@JEEVAN28 try to specify sort order in `/module/MODULE_NAME/Module_name.php`

**Comment by JEEVAN28** [comment]
@vovpff Done, thanks
